examples/basic/static-components/src/App.tsx

```diff
@@ -0,0 +1,20 @@
+import type { Component } from 'solid-js';
+
+import styles from './App.module.css';
+import { CardGrid } from './components/CardGrid';
+import { Navbar } from './components/Navbar';
+import { Title } from './components/Title';
+
+const App: Component = () => {
```

Comment:
I would argue that one common style should be used for Components, I think that export default function FunctionName is also used in the tutorial on site. So export default function App here?